src/v0/destinations/facebook_pixel/data/FBPIXELUserDataConfig.json

```diff
@@ -79,7 +79,10 @@
   {
     "destKey": "name",
     "sourceKeys": ["traits.name", "context.traits.name"],
-    "required": false
```

Comment:
This can happen for any string field (in any destination) where string operations are present in the transformation.
@koladilip Having a schema to validate the payload is the one-stop solution ?

PS: This PR is good to go.